dashboard/src/components/model/System.vue

```diff
@@ -76,10 +84,18 @@ export default class SystemView extends ModelBase {
   lookupElevation() {
     /* istanbul ignore next */
     getElevation(this.parameters.latitude, this.parameters.longitude)
-      .then((elevation: number) => (this.parameters.elevation = elevation))
-      .catch(error => console.log(error.message));
+      .then((elevation: number) => {
+        this.parameters.elevation = elevation;
+        this.elevationError = null;
+      })
+      .catch(() => {
+        this.elevationError =
+          "Elevation information could not be found. Lookup service is only available for sites in the United States.";
```

Comment:
does it work for Hawaii and Alaska? Territories? If not, we should say continental United States.

Reply:
It does. From the service's website: "The 1/3 arc-second dataset covers nearly all the U.S. states and territories, though Alaska has only partial coverage" (https://nationalmap.gov/epqs/) perhaps more specific wording could be used here?

Reply:
this is good enough, thanks.